maxim/tests/test_maxim_core_simple.py

```
@@ -0,0 +1,246 @@
+import os
+import unittest
+from unittest.mock import Mock, patch
```

Ruff (0.11.9)

`unittest.mock.Mock` imported but unused

Remove unused import: `unittest.mock.Mock`

(F401)